### PR TITLE
Add target product (https://issues.jboss.org/browse/JDF-108)

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: helloworld-html5/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Author: Jay Balunas, Burr Sutter, Douglas Campos, Bruno Olivera
 Level: Beginner
 Technologies: CDI, JAX-RS, HTML5
 Summary: Basic HTML5 |Demonstrates the use of CDI 1.0 and JAX-RS using the HTML5 architecture and RESTful services on the backend
+Target Product: EAP
```
Comment (pmuir) [inline helloworld-html5/README.md:7]: This should be WFK, and this change needs submitting to Aerogear

File: helloworld-jsf/README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ Author: Brian Leathem
 Level: Beginner
 Technologies: CDI, JSF, RichFaces
 Summary: Similar to the helloworld quickstart, but with a JSF front end
+Target Product: WFK
```
Comment (pmuir) [inline helloworld-jsf/README.md:7]: This change needs submitting to richfaces quickstarts.

BTW we should consider renaming this to helloworld-richfaces.

File: hibernate3/README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ Author: Bartosz Baranowski
 Level: Intermediate
 Technologies: Hibernate 3
 Summary: Example that uses Hibernate 3 for database access. Compare the code in this quickstart to the _hibernate4_ quickstart to see the changes needed to upgrade to Hibernate 4.
+Target Product: EAP
```
Comment (pmuir) [inline hibernate3/README.md:7]: This doesn't have a target product...

File: kitchensink-html5-mobile/README.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ Author: Jay Balunas
 Level: Intermediate
 Technologies: HTML5
 Summary: Based on kitchensink, but uses HTML5, making it suitable for mobile and tablet computers
+Target Product: EAP
```
Comment (pmuir) [inline kitchensink-html5-mobile/README.md:7]: This should be WFK, and it needs submitting to aerogears quickstarts...

File: wicket-ear/README.md [hunk 5]
```diff
@@ -4,6 +4,7 @@ Author: Ondrej Zizka <ozizka@redhat.com>
 Level: Intermediate
 Technologies: Apache Wicket, JPA
 Summary: Demonstrates how to use the Wicket Framework 1.5 with the JBoss server using the Wicket-Stuff Java EE integration, packaged as an EAR
+Target Product: EAP
```
Comment (pmuir) [inline wicket-ear/README.md:7]: This doesn't have a target product.

File: wicket-war/README.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ Author: Ondrej Zizka <ozizka@redhat.com>
 Level: Intermediate
 Technologies: Apache Wicket, JPA
 Summary: Demonstrates how to use the Wicket Framework 1.5 with the JBoss server using the Wicket-Stuff Java EE integration packaged as a WAR
+Target Product: EAP
```
Comment (pmuir) [inline wicket-war/README.md:7]: This doesn't have a target product.

Comment (sgilda) [inline wicket-war/README.md:7]: Pete, are you saying it should not have a target product?

File: helloworld-html5/README.md (hunk 1 above)
Comment (sgilda) [inline helloworld-html5/README.md:7]: Added issue: https://issues.jboss.org/browse/AEROGEAR-432

Pete, this quickstart is in the wrong branch. It's in master-eap6. It needs to be moved to master-wfk.

File: hibernate3/README.md (hunk 3 above)
Comment (sgilda) [inline hibernate3/README.md:7]: This is in the master-eap6 branch of the https://github.com/jboss-eap/quickstart  repository. Should it be removed?

Is this one just for the community?

File: kitchensink-html5-mobile/README.md (hunk 4 above)
Comment (sgilda) [inline kitchensink-html5-mobile/README.md:7]: Then this quickstart is in the wrong branch. It's in master-eap6. It needs to be moved to master-wfk.

File: wicket-ear/README.md (hunk 5 above)
Comment (sgilda) [inline wicket-ear/README.md:7]: This is in the master-eap6 branch of the https://github.com/jboss-eap/quickstart repository. Should it be removed?

Is this only for community?

File: helloworld-jsf/README.md (hunk 2 above)
Comment (sgilda) [inline helloworld-jsf/README.md:7]: Created JIRA: https://issues.jboss.org/browse/RF-12484

I'm not sure if that's the correct location.

**Comment by sgilda** [comment]
Issued https://github.com/jboss-jdf/jboss-as-quickstart/pull/335 for this fixes.

File: wicket-war/README.md (hunk 6 above)
Comment (pmuir) [inline wicket-war/README.md:7]: Yes

File: hibernate3/README.md (hunk 3 above)
Comment (pmuir) [inline hibernate3/README.md:7]: Currently under discussion, but probably.

File: wicket-ear/README.md (hunk 5 above)
Comment (pmuir) [inline wicket-ear/README.md:7]: Probably only for community, we're still working on this.

File: helloworld-jsf/README.md (hunk 2 above)
Comment (pmuir) [inline helloworld-jsf/README.md:7]: It is :-)

**Comment by rafabene** [comment]
https://github.com/jboss-jdf/jboss-as-quickstart/pull/335 closed

**Pull request merged**